config.go

```diff
@@ -38,4 +49,8 @@ type Config struct {
 
 	// Interfaces will override the interfaces used for queries and answers.
 	Interfaces []net.Interface
+
+	// MulticastConnV6 is used to receive mDNS questions over IPv6. It can be used in conjunction
```

Comment:
This is arguably funky not being part of `Server` but it lets us not break the API for now

Reply:
@edaniels I think it is ok to break the API! Users are pretty low https://pkg.go.dev/github.com/pion/mdns?tab=importedby